input/examples/patient-example-sex-and-gender.xml

```diff
@@ -1,20 +1,24 @@
 <Patient xmlns="http://hl7.org/fhir">
     <id value="example-sex-and-gender"/>
-    <!--   This purpose of this example is to demonstrate the R5 pre-adopted extensions for sex and gender:
-    - Individual Pronouns
+    <!--   This purpose of this example is to demonstrate the use of sex and gender elements:
+    - Gender Harmony Gender Identity (Individual Gender Identity extension)
+    - Gender Harmony Pronouns (Individual Pronouns extension)
+    - and demonstrate the R5 pre-adopted extensions for sex and gender:
```

Comment:
last bullet point isn't adjusted for the PR change. Should say Gender Harmony Recorded Sex or Gender (Person Recorded Sex Or Gender extension)

Reply:
Updated comment

Reply:
Removed